.github/workflows/ios_build.yml

```diff
@@ -0,0 +1,23 @@
+name: KaMPKit iOS
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build
+        uses: sersoft-gmbh/xcodebuild-action@v1
```

Comment:
Is there really no first-party xcodebuild action? I think that was one of the things bugging me last time I looked at actions.

Reply:
Yeah, there's really no first-party xcodebuild action that does what we want. It's pretty lame.

Here are our options:

1. https://github.com/sersoft-gmbh/xcodebuild-action: 20 stars, no signing required
2. https://github.com/Apple-Actions: don't include a simple build action
3. https://github.com/yukiarrr/ios-build-action: 75 stars, codesigning-identity and team-id required (should consider using this for something user-facing w/ full code signing support)
4. https://github.com/devbotsxyz/xcode-build: 1 star, doesn't support iOS, last commit 6 months ago
5. Writing our own action that just running a raw xcodebuild command